### PR TITLE
Automatic refactorings.  These DO need to be reviewed.

### PR DESCRIPTION
**Pull request by robinroos**
Automatic refactorings.  These DO need to be reviewed.  Typically:
* Thread.currentThread().sleep -> Thread.sleep
* Removal of unnecessary types in expressions

### CODE REVIEW

**Comment by robinroos** [comment]
PR build is green.

**Comment by robinroos** [comment]
That's all for tonight.  I'm spending the morning in Birmingham tomorrow and will do more on this repo when I get back to Milton Keynes.

File: src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java [hunk 1]
```diff
@@ -13,15 +13,13 @@
 public class DockerSwarmDiscoveryStrategyFactory implements DiscoveryStrategyFactory {
 
     private static final Collection<PropertyDefinition> PROPERTIES =
```
Comment (bitsofinfo) [inline src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java]: You missed adding `STRICT_DOCKER_SERVICE_NAME_COMPARISON` here

**Comment by bitsofinfo** [comment]
Also please adjust the switch according to: https://github.com/bitsofinfo/hazelcast-docker-swarm-discovery-spi/issues/39

File: src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java (hunk 1 above)
Comment (robinroos) [reply src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/DockerSwarmDiscoveryStrategyFactory.java]: Added that now.

**Comment by robinroos** [comment]
New PropertyDefinition is added to the collection.  PR build is green.

Fix for Issue #39 is on a separate PR #43.

**Pull request merged**